package.json

```
@@ -12,7 +12,7 @@
     "axios": "1.6.8",
     "clsx": "^1.1.1",
     "dotenv-flow": "^3.2.0",
-    "express": "^4.17.2",
+    "express": "4.21.0",
```

Do we mind the minor?:
```suggestion
    "express": "^4.21.0",
```

---

Yes, that's a good point. Already added `^`.